lang/en/local_linkchecker_robot.php

```diff
@@ -81,7 +81,7 @@
 $string['size'] = 'Size';
 $string['url'] = 'URL';
 $string['mimetype'] = 'Mime type';
-$string['broken'] = 'Broken URL';
+$string['brokenlinksandurls'] = 'Broken URL';
```

Comment:
This has broken stuff:

Invalid get_string() identifier: 'broken' or component 'local_linkchecker_robot'. Perhaps you are missing $string['broken'] = ''; in /var/www/vanilla-moodle/local/linkchecker_robot/lang/en/local_linkchecker_robot.php?
line 349 of /lib/classes/string_manager_standard.php: call to debugging()
line 6820 of /lib/moodlelib.php: call to core_string_manager_standard->get_string()
line 110 of /local/linkchecker_robot/report.php: call to get_string()